2 sign up

jaxdid opened the pull request:
closes #2

tcpickard94 committed:
Updated README

tcpickard94 committed:
WIP - started user story 2

Created sign-up form on index page
Created index route
Written first feature test
Disabled database cleaner for time being

jaxdid committed:
Completed User Story 2. Created User model. Added POST route to add sign-up form data to database and creates user session. Added GET route for spaces. Added spaces view. Reenabled Database Cleaner. Removed RSpec-Sinatra from Gemfile.

RPiper93 commented on app/models/User.rb:
```diff
@@ -0,0 +1,13 @@
+  property :first_name, String, required: true
+  property :last_name, String, required: true
+  property :email, String, format: :email_address, required: true, unique: true
+  property :password, String, required: true
```
Shouldn't the password be encrypted? :smile:

jaxdid committed:
Added password encryption

Updated User model to support password encryption within database
Added Rakefile (auto-migrate and auto-upgrade tasks)

jaxdid committed:
Updated Rakefile to include task descriptions

jaxdid committed:
Merged master. Fixed conflicts

russellvaughan commented:
:shipit:
